tools/testing/gatekeeper/required-tests.yaml

```diff
@@ -0,0 +1,36 @@
+required_tests:
+  # Always required tests
+  - Commit Message Check
+required_regexps:
+  # Always required regexps
+
+paths:
+  # Mapping of path (python) regexps to set-of-tests (sort by order of importance)
+  # CI
+  - "^ci/openshift-ci/": []
+  - "^\\.github/workflows/": []
+  # TODO: Expand filters
+  # Documentation
+  #- "\\.rst$": ["build"]
+  #- "\\.md$": ["build"]
+  # Sources
+  #- "^src/": ["static", "build", "test"]
+
+mapping:
+  # Mapping of set-of-tests to required test names and/or test name regexps
```

Comment:
I would simplify this and make each of `test/build/static` only take a list of regexes, since a name can easily be converted into a regex by escaping it.

Reply:
Well that depends on the usage. If we're likely to be using regexps only, then yes, but if there were many individual jobs people wanted to enumerate, then having it as a list should be easier to read and actually safer as it's clearly stated what "items" are expected. (underneath the regexps and names are handled completely differently, individual names can be expected in 1:1 mapping while regexps resolve into unknown amount of items).

With that said I'd prefer keeping it this way for now and once we have some real-world config we could refine, what do you think?

Reply:
sounds reasonable to me.